### PR TITLE
Fix examples from contributing.md

### PR DESCRIPTION
**Pull request by iamandrewluca**

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -40,16 +41,16 @@ Here's an example using all of the possible `feature.add` options:
...
 
-let delegate;
+let delegate: DelegateSubscription | undefined;
 function log() {
 	console.log('✨', <div className="rgh-jsx-element"/>);
 }
 function init(): void {
 	// Events must be set via delegate, unless shortlived
-	delegate = delegate('.btn', 'click', log);
+	delegate = delegateIt('.btn', 'click', log);
```
Comment (fregante) [inline contributing.md:53]: This was fine as `delegate`

Comment (iamandrewluca) [reply contributing.md:53]: it conflicted with `let delegate;`

Comment (fregante) [reply contributing.md:53]: Indeed. Better change that variable than this one then. It was just a random example

Comment (iamandrewluca) [reply contributing.md]: @fregante done!

**Comment by fregante** [comment]
Mersi Andrew ☺️

**Pull request merged**